core/src/main/scala/sbtorgpolicies/libraries.scala

```diff
@@ -51,7 +51,7 @@ object libraries {
     "doobie"                   -> "0.5.0-M9",
     "embedded-redis"           -> "0.6",
     "export-hook"              -> "1.2.0",
-    "finch"                    -> "0.16.0-M3",
+    "finch"                    -> "0.16.0-RC1",
```

Comment:
M3 is more recent than RC1

Reply:
Oh, bad merge. Thanks for pointing out it.